### ISSUE TITLE
Bug on the model 

### ISSUE THREAD

**Issue by spibox** [TShineCN/ChineseBride-And-NewYear-Ghostgirl#1]
it still havent change the model into a chinese girl dress even if i tried 3 of the codes already

**Comment by TShineCN** [comment]
> it still havent change the model into a chinese girl dress even if i tried 3 of the codes already

This mod will conflict with some other mods. And when running this mod alone will not result in errors.
What I currently know is: Diversity (modified audio properties) TooManyEmotes (rewrote the logic about little girl rendering, I don't know why it needs to be rewritten)
I may need a longer time to try to solve these issues, as these mods have all modified the same attribute.

**Comment by spibox** [comment]
I also want to tell you that this is the best ghost girl mod we've ever discovered, the fear when the way she respawns at your back is so loud and terrifying, made me scream at the top of my lungs 🤣🤣🤣🤣

**Comment by TShineCN** [comment]
> I also want to tell you that this is the best ghost girl mod we've ever discovered, the fear when the way she respawns at your back is so loud and terrifying, made me scream at the top of my lungs 🤣🤣🤣🤣

 I really appreciate you like it !!
I feel that the volume and playback logic of the audio still need to be adjusted to make it more .......dark and gloomy? 
Try to minimize the use of very loud volume to scare people.

**Comment by spibox** [comment]
no no no, dont minimize the volume. It's great scaring off people. 🤣

**Comment by cangming99** [comment]
`Added a config entry to disable the girl patch in case of it conflicting with another mod. This patch is to fix the issue with the emote camera seeing the girl while she's "unrendered".`TooManyEmotes resolved this issue in the recent update. 
However, the mod’s sound is indeed a bit too loud, could it be slightly lower?

**Comment by TShineCN** [comment]
> `Added a config entry to disable the girl patch in case of it conflicting with another mod. This patch is to fix the issue with the emote camera seeing the girl while she's "unrendered".`TooManyEmotes resolved this issue in the recent update. However, the mod’s sound is indeed a bit too loud, could it be slightly lower?

Yes, if there is the next update, I will adjust the volume and add some other things~
